lib/core/src/lib/form/services/widget-visibility.service.ts

```diff
@@ -128,7 +128,7 @@ export class WidgetVisibilityService {
         } else if (visibilityObj.rightType === WidgetTypeEnum.field) {
             valueFound = this.getFormValue(form, visibilityObj.rightValue);
         } else {
-            if (moment(visibilityObj.rightValue, 'YYYY-MM-DD', true).isValid()) {
```

Comment:
marked PR as do not merge as this line needs tripple-checking that it works as expected with real data